docs/getting_started.rst

```diff
@@ -0,0 +1,394 @@
+Xarray and Xandas operations in a single pipeline:
+
+.. ipython:: python
+
+    import sgkit as sg
+    ds = sg.simulate_genotype_call_dataset(n_variant=100, n_sample=50, missing_pct=.1)
+
+    # Use `pipe` to apply a single sgkit function to a dataset
+    ds_qc = ds.pipe(sg.variant_stats).drop_dims('samples')
+    ds_qc
+
+    # Show statistics for one of the arrays to be used as a filter
+    ds_qc.variant_call_rate.to_series().describe()
+
+    # Build a pipeline that filters on call rate and computes Fst between two populations
+    (
+        ds
+        # Add call rate and other statistics
+        .pipe(sg.variant_stats)
+        # Apply filter to include variants present across > 80% of samples
+        .pipe(lambda ds: ds.sel(variants=ds.variant_call_rate > .8))
+        # Assign a "cohort" variable that splits samples into two groups
+        .assign(sample_cohort=np.repeat([0, 1], ds.dims['samples'] // 2))
+        # Compute Fst between the groups
+        # TODO: Refactor based on https://github.com/pystatgen/sgkit/pull/260
+        .pipe(lambda ds: sg.Fst(*(g[1] for g in ds.groupby('sample_cohort'))))
+        # Extract the single Fst value from the resulting array
+        .item(0)
+    )
+
+This is possible because sgkit functions nearly always take a ``Dataset`` as the first argument, create new
+variables, and then merge these new variables into a copy of the provided dataset in the returned value.
+See :ref:`dataset_merge` for more details.
+
+Chunked arrays
+~~~~~~~~~~~~~~
+
+Chunked arrays are required when working on large datasets. Libraries for managing chunked arrays such as `Dask Array <https://docs.dask.org/en/latest/array.html>`_
+and `Zarr <https://zarr.readthedocs.io/en/stable/>`_ make it possible to implement blockwise algorithms that operate
+on subsets of arrays (in parallel) without ever requiring them to fit entirely in memory.
+
+By design, they behave almost identically to in-memory (typically Numpy) arrays within Xarray and can be interchanged freely when provided
+to sgkit functions. The most notable difference in behavior though is that operations on chunked arrays are `evaluated lazily <https://tutorial.dask.org/01x_lazy.html>`_.
+This means that if an Xarray ``Dataset`` contains only chunked arrays, no actual computations will be performed
+until one of the following occurs:
+
+- `Dataset.compute <http://xarray.pydata.org/en/stable/generated/xarray.Dataset.compute.html>`_ is called
+- `DataArray.compute <http://xarray.pydata.org/en/stable/generated/xarray.DataArray.compute.html>`_ is called
+- The ``DataArray.values`` attribute is referenced
+- Individual dask arrays are retrieved through the ``DataArray.data`` attribute and forced to evaluate via `Client.compute <https://distributed.dask.org/en/latest/api.html#distributed.Client.compute>`_, `dask.array.Array.compute <https://tutorial.dask.org/03_array.html#Example>`_ or by coercing them to another array type (e.g. using np.asarray)
```

Comment:
Quote `np.asarray` and possibly link to its doc.

Reply:
https://github.com/pystatgen/sgkit/commit/f7f5a039412fc6b4c29621d283c1a5738b746521#diff-9bc8bf6e8d9db020cefce1fbc0426795R303